BRANDING.MD

```diff
@@ -43,7 +43,7 @@ Suggested templates:
 * The Knative name may not be used or registered in a manner that would cause
   confusion as to Google’s affiliation or endorsement (as the trademark owner).
```

Comment:
This line needs to be update _(as the trademark owner)._ CNCF is now the new trademark owner

Reply:
Good catch - will update